src/EnergyPlus/PluginManager.cc

```diff
@@ -596,6 +926,18 @@ namespace PluginManagement {
 #endif
     }
 
+    PluginManager::~PluginManager()
+    {
+#if LINK_WITH_PYTHON
+        //(*EP_Py_FinalizeEx)();
```

Comment:
Calling finalize was causing some big problems.  But after thinking this through, we shouldn't need to do that.  The only time Python is initialized is during PythonPlugin simulations.  Those simulations are only available when executing EnergyPlus as an executable.  When calling EnergyPlus this way, the entire thread and all contents are deleted when EnergyPlus finishes.  No need to finalize, because we won't be initializing again here.